### DISCUSSION TITLE
ارور عدم فعال بودن وارپ warp

### DISCUSSION THREAD

**Discussion by skydarklord** [hiddify/Hiddify-Manager#3881] [Bug باگ: please use issue]
با عرض سلام و تبریک

 سال نو به همه دوستان گرامی 
2 شب پیش با عدم اتصال وی پی ان متوجه مشکلی شدم که باعث شد هیدیفای منیجر رو به نسخه 10.12.1 آپدیت کنم . بعد از آپدیت همچنان مشکل وجود داشت و کانفیگها کار نمی کردند . با غیر فعال کردن وارپ مشکل تا حدودی رفع شد و در حال حاظر مستقیم به آی پی سرور متصل هستم . کلید توسعه کلود فلر و وارپ پلاس رو عوض کردم و بعد اعمال تغییرات این ارور در انتهای لاگ نمایش داده شد . 
جالب اینجاست با فعال سازی وارپ و غیر فعال سالزی در هر دو حالت این ارور نمایش داده میشه .

ممنون میشم راهنمایی کنید

 .

####75####Installing...####Warp####
Error: Key not found: warp_enable
===========================================================
===run.sh other/warp
Checking WARP ...
Checking WARP ...
mv: cannot stat 'wgcf-account.toml': No such file or directory
Checking WARP ...
mv: cannot stat 'wgcf-account.toml': No such file or directory
Generating WARP config...
Removing IPV6 from WARP...
Starting WARP.... checking real connectivitiy
WARP is not working!
Checking real connectivitiy again!
WARP is not working!
!!!!!!!!!!!!!!! WARP ERROR
===========================================================
####85####Installing...####Wireguard####
===========================================================
===run.sh other/wireguard
===========================================================
####90####Installing...####HiddifyCli####
Error: Key not found: hiddifycli_enable
===============================================

**Comment by MrLlnux** [comment]
منم همین مشکل رو دارم

**Comment by alintjm** [comment]
same

**Comment by mhmd-android** [comment]
same
